tests/test_portal.py

```diff
@@ -168,21 +171,25 @@ def test_message_sent(self):
 
         # confirm styling unicode functions
         body = message.style_message(message.body)
-        self.assertTrue(u'DOCTYPE' in body)
-        self.assertTrue(u'style' in body)
-        self.assertTrue(isinstance(body, unicode))
+        assert u'DOCTYPE' in body
+        assert u'style' in body
+        if sys.version_info[0] < 3:
```

Comment:
Try using `basestring` for your `isinstance()` check, then you won't need version-dependent code